tests/unit/core/handler-test.ts

```diff
@@ -174,6 +174,19 @@ module('Unit | core/handler', function (hooks) {
     assert.ok(handlerTriggerEventSpy.calledOnceWithExactly('remove-row'));
   });
 
+  test('Handler/mutateRows', async function (assert: Assert) {
```

Comment:
`Handler#mutateRows` to be coherent with other

Reply:
```suggestion
  test('Handler#mutateRows', async function (assert: Assert) {
```